src/Commands/ScaffoldExtensionCommand.php

```diff
@@ -59,16 +59,15 @@ public function runScaffoldExtensionsJob(string $site_info = '', string $job_id
         $site_arr = Helpers\UtilityFunctions::decypherSiteInfo($site_info);
         $site_id = $site_arr['id'];
         $site_env = $site_arr['env'];
-        $site = $this->getSite($site_id);
-        $env = $site->getEnvironments()->get($site_env);
-
         // Bail if getSiteById doesn't exist.
-        if (! method_exists($this, 'getSiteByID')) {
+        if (! method_exists($this, 'getSiteById')) {
```

Comment:
We could also specify Terminus 3.2 as the required minimum version for this plugin:

https://github.com/pantheon-systems/terminus-addons-installer-plugin/blob/main/composer.json#L43

`^3.2 || ^4` is valid.